.stylelintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "plugins": ["stylelint-a11y"],
+  "extends": [
+    "stylelint-config-standard",
+    'stylelint-a11y/recommended'
+  ],
+    "rules": {
```

Comment:
minor nit: but indenting looks a little off.

Actually, does **stylelint** expose a JS file for config files, kind of like [_eslintrc.js_](https://github.com/ProjectEvergreen/greenwood/blob/master/.eslintrc.js)?  (e.g. _.stylelintrc.js_?)

If the file ends in .js, then we can get linting on this config file too!  (which in theory would auto catch any minor formatting like indention 🥳 )